### PR TITLE
Obstacles

### PR DESCRIPTION
**Pull request by hmyeh**

### CODE REVIEW

File: core/src/main/java/com/group5/core/controllers/Spawner.java [hunk 1]
```diff
@@ -1,8 +1,93 @@
...
+		return floor.getX() + floor.getTexture().getWidth();
+	}
+	
+	/**
+	 * Method to get the player position.
+	 * @return the x coordinate of the player.
+	 */
+	public float getPlayerPosition(){
+		Player player = null;
+		for(WorldObject w : world.getObjects()){
```
Comment (mwolting) [inline core/src/main/java/com/group5/core/controllers/Spawner.java]: World has a getPlayer method that is more efficient than this

**Comment by mwolting** [comment]
Works for me, provided you fix the getPlayer thing

**Comment by hmyeh** [comment]
The getPlayer thing is fixed

**Comment by levilime** [comment]
:+1:

**Comment by levilime** [comment]
fixes #6

**Comment by levilime** [comment]
fixes #9

**Pull request merged**